zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/PublishingQueueCollectionDetails.java

```diff
@@ -0,0 +1,60 @@
+package com.github.onsdigital.zebedee.api;
+
+import com.github.davidcarboni.restolino.framework.Api;
+import com.github.onsdigital.zebedee.exceptions.ZebedeeException;
+import com.github.onsdigital.zebedee.json.CollectionDetail;
+import com.github.onsdigital.zebedee.session.model.Session;
+import com.github.onsdigital.zebedee.util.ZebedeeCmsService;
+import org.eclipse.jetty.http.HttpStatus;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.GET;
+import java.io.IOException;
+
+@Api
+public class PublishingQueueCollectionDetails {
+
+    private static ZebedeeCmsService zebedeeCmsService = ZebedeeCmsService.getInstance();
+
+    /**
+     * Retrieves a CollectionDetail object at the endpoint /PublishingQueueCollectionDetails/[CollectionName]
+     *
+     * @param request  This should contain a X-Florence-Token header for the current session
+     * @param response <ul>
+     *                 <li>If no collection exists:  {@link HttpStatus#NOT_FOUND_404}</li>
+     *                 </ul>
+     * @return the CollectionDetail.
+     * @throws IOException
+     */
+    @GET
+    public CollectionDetail get(HttpServletRequest request, HttpServletResponse response)
+            throws IOException, ZebedeeException {
+
+        com.github.onsdigital.zebedee.model.Collection collection = Collections
```

Comment:
Is there a reason we're not importing this like the statements at the top of the file?

Reply:
I think it has been done elsewhere due to name clashes, where the same class name is used in two different namespaces. I don't think it's required here though, so I will do the import 👍